karma.conf.js

```diff
@@ -20,7 +20,9 @@ module.exports = function(config) {
       'node_modules/jasmine-promises/dist/jasmine-promises.js',
       'node_modules/webcomponentsjs/full.js',
       'dist/js/patternfly.js',
-      'src/**/*.spec.js'
+      'src/**/*.spec.js',
+      'dist/css/patternfly.css',
+      'dist/css/patternfly-webcomponents.css'
```

Comment:
Do not include them at all. You probably need CSS for transitions, instead of depending upon the events, manually call the handlers